src/CattyTests/Formula/FormulaFunctionsTest.swift

```diff
@@ -0,0 +1,190 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+import XCTest
+
+@testable import Pocket_Code
+
+final class FormulaFunctionsTest: XCTestCase {
+    var formulaManager: FormulaManager!
+    var spriteObject: SpriteObject!
+
+    let EPSILON = 0.001
+
+    override func setUp() {
+        let screenSize = Util.screenSize(true)
+        formulaManager = FormulaManager(sceneSize: screenSize)
+        spriteObject = SpriteObject()
+    }
+
+    func getFormulaForFunction(tag: String, leftValue: String, rightValue: String?) -> Formula {
+        let leftElement = FormulaElement(elementType: ElementType.NUMBER, value: leftValue, leftChild: nil, rightChild: nil, parent: nil)
+        var rightElement: FormulaElement?
+
+        if rightValue != nil {
```

Comment:
Just to let you know: please prefer "if let" in order to remove to force-unwrap.

Reply:
Yes please update this!